src/server/worker/pipeline/service/service.go

```diff
@@ -2,6 +2,7 @@ package service
 
 import (
 	"context"
+	"fmt"
```

Comment:
I imagine this is breaking the build, but otherwise LGTM

Reply:
:joy: Whoops, thanks!!